e2e/various-suite/graph-auto-migrate.spec.ts

```diff
@@ -12,7 +12,7 @@ describe('Auto-migrate graph panel', () => {
     cy.contains(DASHBOARD_NAME).should('be.visible');
     cy.contains('uplot-main-div').should('not.exist');
 
-    e2e.flows.openDashboard({ uid: DASHBOARD_ID, queryParams: { '__feature.autoMigrateOldPanels': true } });
+    e2e.flows.openDashboard({ uid: DASHBOARD_ID, queryParams: { '__feature.autoMigrateGraphPanel': true } });
```

Comment:
i guess the intention here is to exclude the old graph panel from `autoMigrateOldPanels`, not simply add another flag that only does `Graph (old)`?

Reply:
ah that's a fair point, and would also get around complexity of migrating instance feature toggles accordingly 🤔 I can modify this so that `autoMigrateOldPanels` still has same functionality

Reply:
just pushed up a commit to address this 👍